src/include/platform_support.h

```diff
@@ -45,7 +45,7 @@ int platform_hwversion(void);
 void platform_nrst_set_val(bool assert);
 bool platform_nrst_get_val(void);
 bool platform_target_get_power(void);
-void platform_target_set_power(bool power);
+void platform_target_set_power(const bool power);
```

Comment:
clang-tidy complains if you put const here in the declaration. It's only required on the definition.

Reply:
Ah, so that's why it didn't have it, TIL